gcloud/streaming/http_wrapper.py

```diff
@@ -13,8 +12,8 @@
 
 import httplib2
 import six
-from six.moves import http_client
-from six.moves.urllib import parse
+from six.moves import http_client   # pylint: disable=F0401
+from six.moves.urllib import parse  # pylint: disable=F0401
```

Comment:
Sad. I had heard the `pylint` folks would get better at supporting `six`, but then again, they told us they'd do a 1.5 release around July 15-17 and it still hasn't been released.